### PR TITLE
Add a quick and dirty build with bazel.

### PR DESCRIPTION
**Pull request by alercah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,10 +13,16 @@ in excess of the resource limits of the machine running the build.
...
+build the repository using the included `BUILD` files, Bazel will fetch the
+protobuf compiler and googleapis automatically.
 
 ## Using the APIs
 
-We currently do not have a build system in this repository. All the APIs are
-written in [gRPC](grpc.io), and can be compiled to the language of your choice
-using the protobuf compiler.
+The repository contains `BUILD` files to build the protobuf library with
+[Bazel][https://bazel.build/]. If you wish to use them with your own project in
```
Comment (ola-rozenfeld) [inline README.md]: the link should be in () rather than []

File: build/bazel/remote/execution/v0_2/BUILD [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package(default_visibility = ["//visibility:public"])
+
+# load("@protobuf//:protobuf.bzl", "proto_library")
```
Comment (ola-rozenfeld) [inline build/bazel/remote/execution/v0_2/BUILD]: Wait, why isn't this needed?

Comment (alercah) [reply build/bazel/remote/execution/v0_2/BUILD]: The `proto_library` rule is builtin to Bazel; I originally thought I had to load it (it does depend on the protoc repository though, which must be explicitly specified in `WORKSPACE`). I'll remove this line.

File: README.md (hunk 1 above)
Comment (alercah) [reply README.md]: Fixed.

**Pull request merged**